### DISCUSSION TITLE
Connection refused error when executing encrypted Pyarmor code (ubuntu)

### DISCUSSION THREAD

**Discussion by nrtoribio** [dashingsoft/pyarmor#1346] [General]
I am encountering an issue while attempting to execute encrypted code using Pyarmor on a Linux machine. Here are the details of the problem:

License: We have a Pyarmor Group license.
Encryption Process: On a ubuntu22 with Python 3.10 and pyarmor 8.2.4, we encrypted our code using the command "pyarmor gen hello_world.py". This process successfully generated the dist folder, containing the encrypted code and the pyarmor_runtime_006024 folder.
Transfer to another machine: We copied the dist folder generated by Pyarmor onto another ubuntu22 machine for execution.
Error Message: However, when we try to execute the code with Python 3.10 using the command "python file.py", we encounter the following error:
![connection_refused_error](https://github.com/dashingsoft/pyarmor/assets/137907517/5381900a-4f1b-434f-bdae-41f59dcd1abe)

We are seeking assistance in understanding the cause of this error and any potential solutions. I would appreciate any guidance or insights that can help resolve this issue. Thank you!

**Comment by jondy** [comment]
Are you sure the obfuscated scripts has no expired date? This error only occurs when the obfuscated has expired date.

Please make sure no expired date set by `pyarmor cfg`
